### PR TITLE
fix: add new Bazel workflow to docs

### PR DESCRIPTION
**Pull request by miraleung**
This will work once [this PR](https://github.com/googleapis/googleapis/pull/620) is in.

### CODE REVIEW

**Review by xiaozhenliu-gg5 — APPROVED** [review]
Thanks for doing this!

**Comment by xiaozhenliu-gg5** [comment]
One question when following the development doc, that I cloned the showcase proto and copied them to googleapis/. Since it does not have BUILD.bazel in showcase/v1beta1, so I created a new one and copied the script to it, while I got error like:
```
/Users/xiaozhenliu/googleapis/google/showcase/v1beta1/BUILD.bazel:1:1: file '@com_google_googleapis_imports//:imports.bzl' does not contain symbol 'java_microgenerator_gapic_assembly_gradle_pkg'
```
Am I missing something?

**Comment by miraleung** [comment]
The error is because [this PR](https://github.com/googleapis/googleapis/pull/620) isn't merged yet.

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -18,33 +18,58 @@ below are temporary and better ones will be coming.
 1.  Clone [googleapis](https://github.com/googleapis/googleapis) and
     [gapic-showcase](https://github.com/googleapis/gapic-showcase/) and install
     protoc.
+
 2.  Copy the protos from Showcase into googleapis/google/showcase.
 
     ```sh
     cp gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta
     ```
```
Comment (xiaozhenliu-gg5) [inline DEVELOPMENT.md:26]: `cp -r gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta1`

**Pull request merged**